public/js/twenty.js

```diff
@@ -25,6 +29,31 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
+  const notify = () => {
+    if (Notification.permission !== 'granted') {
+      return;
+    }
+
+    let message;
+
+    if (mode === 'twenty') {
+      if (!stop) {
+        // can't see the notification if they're looking away!
+        return;
+      }
+
+      message = 'Time to look away!';
+    } else {
+      if (stop) {
+        message = 'Time to stand up!';
+      } else {
+        message = 'Time to sit down!';
+      }
```

Comment:
can you make it play "up" and "down" from https://www.youtube.com/watch?v=qJdMjRHRLfg?